### PR TITLE
changed image labels from visually-hidden to hidden

### LINKED ISSUE #165: Image label marked visually-hidden
`    label: visually_hidden
`

The field label is set to visually hidden, which means screen readers read out "Image" on every image, which is unnecessary. 
We can set this to Hidden

### PR DESCRIPTION
**Pull request by bisd**
Please review this pull request.

### CODE REVIEW

File: modules/localgov_media/config/optional/core.entity_view_display.media.image.localgov_event_thumbnail.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+langcode: en
```
Comment (markconroy) [inline modules/localgov_media/config/optional/core.entity_view_display.media.image.localgov_event_thumbnail.yml]: I think this file needs to be removed from this PR and added to localgov_events repo instead.

File: modules/localgov_media/config/optional/core.entity_view_display.media.image.localgov_newsroom_teaser.yml [hunk 2]
```diff
@@ -14,16 +14,19 @@ bundle: image
 mode: localgov_newsroom_teaser
```
Comment (markconroy) [inline modules/localgov_media/config/optional/core.entity_view_display.media.image.localgov_newsroom_teaser.yml:14]: I _think_ this file should be in the localgov_news repo. I'm surprised that we seem to have this image style in localgov_core

**Review by markconroy — CHANGES_REQUESTED** [review]

File: modules/localgov_media/config/optional/core.entity_view_display.media.image.localgov_event_thumbnail.yml (hunk 1 above)
Comment (bisd) [reply modules/localgov_media/config/optional/core.entity_view_display.media.image.localgov_event_thumbnail.yml:1]: Thanks @markconroy  

Does that mean I should:

i) re-submit this pull request without this file?
ii) create a new pull request with this file in web/modules/contrib/localgov_events/config/optional ?

File: modules/localgov_media/config/optional/core.entity_view_display.media.image.localgov_newsroom_teaser.yml (hunk 2 above)
Comment (bisd) [reply modules/localgov_media/config/optional/core.entity_view_display.media.image.localgov_newsroom_teaser.yml:14]: Thanks @markconroy is there a way for me to establish which repo this file should be in, as it is in the localgov_core repo as of now?

Comment (msayoung) [reply modules/localgov_media/config/optional/core.entity_view_display.media.image.localgov_newsroom_teaser.yml:14]: Hey both - it's been in this repo for 3 years, so lets create another issue if we're going to remove it.

File: modules/localgov_media/config/optional/core.entity_view_display.media.image.localgov_event_thumbnail.yml (hunk 1 above)
Comment (bisd) [reply modules/localgov_media/config/optional/core.entity_view_display.media.image.localgov_event_thumbnail.yml]: This file has been removed from this pr with the latest commit. I have created another issue to edit the file in the localgov_events repo:
https://github.com/localgovdrupal/localgov_events/issues/94

Comment (bisd) [reply modules/localgov_media/config/optional/core.entity_view_display.media.image.localgov_event_thumbnail.yml]: The file in the localgov_events repo is already set to "Hidden" so I have closed the issue, deleted the branch and closed the pull request I created for it in the localgov_events repo.

File: modules/localgov_media/config/optional/core.entity_view_display.media.image.localgov_newsroom_teaser.yml (hunk 2 above)
Comment (bisd) [reply modules/localgov_media/config/optional/core.entity_view_display.media.image.localgov_newsroom_teaser.yml:14]: A new issue for relocating this file has been created [here](https://github.com/localgovdrupal/localgov_core/issues/167)

**Comment by finnlewis** [comment]
Leaving this for @markconroy to review again

File: modules/localgov_media/config/optional/core.entity_view_display.media.image.localgov_newsroom_teaser.yml (hunk 2 above)
Comment (markconroy) [reply modules/localgov_media/config/optional/core.entity_view_display.media.image.localgov_newsroom_teaser.yml:14]: If it's currently in this repo, let's leave it in this repo. Maybe there's a reason it was put here originally.

**Review by markconroy — APPROVED** [review]

**Pull request merged**